site/docs/docs/components/pagination.md

```diff
@@ -129,41 +136,42 @@ Vue 3: [component source](https://github.com/AgnosticUI/agnosticui/blob/master/a
 
 ```html
 <script>
-  import 'agnostic-svelte/css/common.min.css';
+  import "agnostic-svelte/css/common.min.css";
   import { usePagination } from "agnostic-helpers/dist/index.esm";
   import { Pagination } from "agnostic-svelte";
 
   // Offset describes how many siblings besides current (must be 1 | 2)
   // Example of offset of 1: [1][2(current)][3]...[50]
   // Example of offset of 2: [1][2][3(current)][4][5]...[50]
-  const paging = usePagination({ offset: 1 });
+  // Pagination by default uses a generator with an offset of 1
+  // but we are including the use of a custom paging generator here as an example
+  const paging = usePagination({ offset: 2 });
 
-  // currentPage is the "dependency" that triggers reactive `paginationPages`
   let currentPage = 1;
-  $: paginationPages = paging.generate(currentPage, 50);
 
-  const onPageUpdated = async (pageNumber) => {
-    console.log("onPageUpdated called with page: ", pageNumber)
-    // This will trigger paginationPages to update itself above
-    currentPage = pageNumber;
-  };
-  
+  // Using $: here declares a reactive block, this code will run whenever
+  // currentPage is given a new value
+  $: console.log(`Current page is: ${currentPage}`);
+
   const paginationArgs = {
-    totalPages: 50,
-    onPageChange: onPageUpdated,
+    total: 50,
+    pageGenerator: paging,
     navigationLabels: {
       previous: "Previa",
       next: "Siguiente",
       first: "Primera",
       last: "Última",
     },
     ariaLabel: "Pagination",
-    justify: "center"
-  }
-
+    justify: "center",
+  };
 </script>
-<Pagination {...paginationArgs} current="{currentPage}" pages="{paginationPages}" />
+
+<!-- Using bind: here we establish a two way connection with the 'current' property -->
```

Comment:
I'd swap connection^binding just to be idiomatic but I really liked the comment otherwise 👍🏽

Reply:
Yeah I'll swap this out and look into my eslint settings